index.html

```diff
@@ -205,9 +205,9 @@ <h2>
           </p>
           <p>
             Pointer lock MUST fail unless the <a>target</a>'s
-            <a>shadow-including root</a> is the <a>active document</a> of a
-            [=Document/browsing context=] which is (or has an <a>ancestor
-            browsing context</a> which is) in focus by a window which is in
+            <a>shadow-including root</a> is the [=active document=] of a
```

Comment:
@marcoscaceres Any clue why ReSpec is still not happy with this PR (see "Generate static HTML" step [here](https://github.com/w3c/pointerlock/actions/runs/7713337739/job/21022995725?pr=88))?  Perhaps the symbols are not exposed in the source specs?  Perhaps I need to "include" other spec definitions here somehow?